gemma-core/src/main/java/ubic/gemma/model/common/description/Characteristic.java

```diff
@@ -44,6 +45,7 @@ public class Characteristic extends AbstractDescribable implements Serializable
      */
     private String originalValue = null;
     private String value;
+    @Nullable
```

Comment:
This is related to #530 as we attempt to identify areas in the code that might be assuming that value URI is empty instead of null.